components/match2/wikis/rainbow_six/match_summary.lua

```diff
@@ -595,7 +595,7 @@ function CustomMatchSummary._getOppositeSide(side)
 	return 'atk'
 end
 
-function CustomMatchSummary._createCheckMark(isWinner)
+function CustomMatchSummary._createCheckMark(isWinner, isLeft)
```

Comment:
```suggestion
function CustomMatchSummary._createCheckMark(isWinner, position)
```

And up top add 

```lua
local _POSITION_LEFT = 1
local _POSITION_RIGHT = 2
```

And then change the call and the usage of course

Reply:
Implemented as suggested. Also applied the same change for older code that was using the similar isLeft logic.

Reply:
That's great!